app/controllers/news_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class NewsController < ApplicationController
+  before_action :set_article, only: %i[show]
+  before_action :set_site
+
+  def index
+    @articles = Article
+      .for_site(@site)
+      .published
+      .order(:published_at)
```

Comment:
we have a scope for this - `by_publish_date`

also needs pagination i think from now. so either make a new ticket or do it as part of this one, i think its essential